package.json

```diff
@@ -22,8 +22,8 @@
     "lint": "eslint packages scripts"
   },
   "devDependencies": {
-    "eslint": "4.19.1",
-    "lerna": "2.11.0"
+    "eslint": "5.16.0",
+    "lerna": "3.15.0"
```

Comment:
Tbh, I'm not so confident about moving lerna to v3. Can we keep it at v2?

Reply:
Let me check again. It was working locally though.

Recreated the PR from correct branch btw => https://github.com/adidas/js-linter-configs/pull/16

Reply:
I tried to update it some time ago and it did not work because the command line has changed a bit.